client/src/inside/components/page/EhrPageTable.vue

```diff
@@ -84,6 +85,11 @@ export default {
     proceedClearAllData () {
       console.log('EhrPageTable clearAllData ', this.tableDef)
       this.ehrHelp.clearTable(this.tableKey)
+      if (this.pageDataKey === 'medicationOrders') {
+        const helper = new MarHelper(this.ehrHelp)
+        helper.clearAllData()
+      }
```

Comment:
Nice except I worry about string constants like 'medictionOrders' being used to identify the page.  Place a constant into the MarHelper with the page key and use that constant.  Look for other cases where 'medicationOrders' is used and replace with constant. Then comment the const to say its the page key for the medications orders page.

Reply:
I have implemented this by reusing a constant which had been implemented in mar-helper (`MED_ORDERS_PAGE_KEY`) and implemented a different wrapper method in `mar-helper` called `triggerActionByPageKey`. Please, let me know your thoughts on it. Thank you!